doc/user/content/sql/system-catalog.md

```diff
@@ -262,56 +262,56 @@ Field             | Type       | Meaning
 `sent`            | [`bigint`] | The number of messages sent.
 `received`        | [`bigint`] | The number of messages received.
 
-### `mz_worker_materialization_dependencies`
+### `mz_worker_compute_dependencies`
 
-The `mz_worker_materialization_dependencies` source describes the dependency structure
+The `mz_worker_compute_dependencies` source describes the dependency structure
 between each [dataflow] and the sources of their data. To create a complete dependency
 structure, the `import_id` column includes other dataflows.
 
 Field      | Type       | Meaning
 -----------|------------|--------
-`export_id`| [`text`]   | The ID of the object that created the dataflow. Corresponds to [`mz_materializations.object_id`](#mz_materializations).
-`import_id`| [`text`]   | The ID of the storage source. Corresponds to [`mz_sources.id`](#mz_sources) or [`mz_tables.id`](#mz_tables) or [`mz_materializations.object_id`](#mz_materializations).
+`export_id`| [`text`]   | The ID of the object that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).
+`import_id`| [`text`]   | The ID of the storage source. Corresponds to [`mz_sources.id`](#mz_sources) or [`mz_tables.id`](#mz_tables) or [`mz_compute_exports.export_id`](#mz_compute_exports).
 `worker_id`| [`bigint`] | The ID of the worker thread hosting the dataflow.
 
-### `mz_materialization_frontiers`
+### `mz_compute_frontiers`
 
-The `mz_materialization_frontiers` view describes the frontier for each
+The `mz_compute_frontiers` view describes the frontier for each
 [dataflow] in the system across all workers. The frontier describes the earliest
 timestamp at which the output of the dataflow may change; data prior to that
 timestamp is sealed.
 
 For per-worker frontier information, see
-[`mz_worker_materialization_frontiers`](#mz_worker_materialization_frontiers).
+[`mz_worker_compute_frontiers`](#mz_worker_compute_frontiers).
 
-Field        | Type       | Meaning
--------------|------------|--------
-`object_id ` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_materializations.object_id`](#mz_materializations).
+Field        | Type            | Meaning
+-------------|-----------------|--------
+`export_id ` | [`text`]        | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).
 `time`       | [`mztimestamp`] | The next timestamp at which the materialization may change.
```

Comment:
We are still referring here to "the materialization". Perhaps rephrase to "the output"?